src/_pytest/_io/terminalwriter.py

```diff
@@ -32,7 +32,7 @@ def should_do_markup(file: TextIO) -> bool:
         and file.isatty()
         and os.environ.get("TERM") != "dumb"
         and not (sys.platform.startswith("java") and os._name == "nt")
```

Comment:
Seems fine to me to remove this if we don't support Jython anymore - looks like it was originally added in https://github.com/pytest-dev/py/commit/536252cb2e661558bf9d9092a6036a42d31c46ad some 10 years ago :laughing:

Reply:
Removed!